Src/BootCamp.Chapter/CredentialsManager.cs

```diff
@@ -0,0 +1,168 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace BootCamp.Chapter
+{
+    internal class CredentialsManager
+    {
+        private const string CredentialsPath = @"..\..\..\Database\Credentials.txt";
+
+        public CredentialsManager()
+        {
+            if (!File.Exists(CredentialsPath))
+            {
+                File.WriteAllText(CredentialsPath, "");
+            }
+        }
+
+        public void Register()
+        {
+            if (RegisterUser())
+            {
+                Console.WriteLine("You have successfully registered.\n");
+            }
+            else
+            {
+                Console.WriteLine("Sorry, your registration was unsuccessful.  The username is already taken.\n");
+            }
+        }
+
+        private bool RegisterUser()
+        {
+            const string CarriageReturn = "\r\n";
+            const string Delimiter = ",";
+
+            var user = GetNameAndPassword();
+
+            if (IsNameRegistered(user.Username))
+            {
+                return false;
+            }
+
+            var encodedPassword = new Cryptography().Encode(user.Password);
+            File.AppendAllText(CredentialsPath, user.Username);
+            File.AppendAllText(CredentialsPath, Delimiter);
+            File.AppendAllText(CredentialsPath, encodedPassword);
+            File.AppendAllText(CredentialsPath, CarriageReturn);
+
+            return true;
+        }
+
+        public bool Login()
+        {
+            var user = GetNameAndPassword();
+
+            if (AuthenticateUser(user))
+            {
+                Console.WriteLine("You have sucessfuly logged in.\n");
+                return true;
+            }
+
+            Console.WriteLine("Sorry, we don't recognize you.\n");
+            return false;
+        }
+
+        private Credentials GetNameAndPassword()
```

Comment:
You don't get credentials. You ReadCredentials (from console)

Reply:
And ofc- instead of many words, use your domain language. PasswordAndUsername = Credentials.

Reply:
Changed the wording to use Credentials instead of PasswordAndUsername for function name.  I appreciate very much whenever you point this out.  Thank you.